app.js

```diff
@@ -26,38 +27,73 @@ function getRandomItem(arr) {
     return arr[index];
 }
 
+// getRandomItem(hidingPlaces);
+
 function handleGuess(userGuess, correctSpot) {
-    // should reset the styles
+    // console.log(userGuess);
+    // console.log(correctSpot);
 
+    // should reset the styles
+    resetStyles();
+    
     // then increment the guesses
-
+    totalGuesses++;
+    
     // then grab the appropriate container element for the correct guess from the DOM
+    // const correctContainer = correctSpot + "-container";
+    // console.log(correctContainer);
 
+    const correctContainerEl = document.getElementById(correctSpot + "-container");
+    console.log(correctContainerEl);
+
+    
     // then add the face class to that element so that the face shows up
+    correctContainerEl.classList.add("face");
 
     // then if the user guess is correct, increment the correct guesses
+    if (userGuess === correctSpot){
+        correctGuesses++;
+    };
```

Comment:
The semicolon on line 56 is unnecessary, that is why you are getting an eslint error that is making github angry.

Reply:
Thanks! Still not totally sure when i need a semicolon and when i dont